packages/react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -181,6 +181,10 @@ export const getDetailsHeaderStyles = (props: IDetailsHeaderStyleProps): IDetail
       },
     ],
 
+    groupExpanderContainer: {
```

Comment:
If we use `display: contents`, don't we risk the screen readers ignoring the element in the accessibility tree? I thought that at this point, the only thing the parent element can do is define CSS variables.